Resources/skeleton/app/SymfonyRequirements.php

```diff
@@ -778,4 +780,21 @@ protected function getRealpathCacheSize()
                 return (int) $size;
         }
     }
+
+    /**
+     * Defines PHP required version from Symfony version.
+     *
+     * @return string The PHP required version
+     */
+    protected function getPhpRequiredVersion()
+    {
+        $composerConfig = json_decode(file_get_contents(__DIR__.'/../composer.json'), true);
+        if (array_key_exists('symfony/symfony', $composerConfig['require'])) {
```

Comment:
Then, check `symfony/http-kernel` as this is a requirement of this bundle, so it will always be installed.